packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageV2.stories.ts

```diff
@@ -35,7 +35,7 @@ export const DemoVersion2: Story = {
     },
   },
   parameters: {
-    stack: 'mock',
+    stack: 'production',
```

Comment:
This should fix the many bugs that were created because users were looking at cached results from the mock service